docs/running-a-node/rocksdb-ledger-backend.md

```diff
@@ -0,0 +1,82 @@
+The node ledger currently uses LMDB (Lightning memory-mapped database) by default as the data store. As of _v20+_ the option to use RocksDB becomes available as an experimental option.
+This document will not go into much detail about theses key-value data stores as there is a lot of information available online.
+It is anticipated that bootstrapping will be slower using RocksDB during the initial version at least, but live traffic should be faster due to singluar writes being cached in memory and flushed to disk in bulk.
+
+Using RocksDB requires a few extra steps as it is an externally required dependency which requires a recent version of RocksDB so older repoistories may not be sufficient, it also requires `zlib`. If using the docker node, can skip to 3.:  
```

Comment:
"..which requires a recent version of RocksDB, so older..." (add comma) and spelling: "repoistories" > "repositories"